src/Server.h

```diff
@@ -202,6 +215,9 @@ class cServer
 	size_t m_MaxPlayers;
 	bool m_bIsHardcore;
 
+	/** Map of protocol version to Forge mods for that version. */
+	std::map<UInt32, AStringMap> m_ForgeModsByVersion;
```

Comment:
Since you're already touching this file due to @Seadragon91 's review, might be worth detailing what this is:
```cpp
/** Map of protocol version to Forge mods (map of ModName -> ModVersionString) for that version. */
```
Perhaps add it to the public getter, too.